### DISCUSSION TITLE
lazyload 이미지 로딩이 느립니다.

### DISCUSSION THREAD

**Discussion by bbingse** [pronist/hELLO#236] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: [e.g. windows] windows
- 브라우저: [e.g. chrome, safari] chorme
- 버전: [e.g. 22] 125.0.6422.78
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): [e.g 3.4.0] 4.2.2

**설명**
버그가 어떻게 발생하는지 알려주세요. 대략적인 버그의 설명을 적어주시면 됩니다.
[e.g. 코드 하이라이팅이 동작하지 않습니다.]

이미지를 많이 업로드하는 블로그입니다.
다량의 이미지를 업로드하고 스크롤할시
몇몇 이미지가 불러오는 속도가 너무 늦어서
오랜시간 공백으로 유지되는 경우가 있습니다.
조금 더 빠른시점에서 이미지를 불러오는건 불가능할까요?

**발생 과정**
버그가 어떤 과정을 통해 발생하는지 구체적으로 적어주세요.
[e.g. 테마 변경 버튼을 통해 화이트 모드로 변경시 코드 하이라이팅의 색상이 변하지 않습니다. ...]

**스크린샷 및 링크**
버그가 발생한 상황을 가능하다면 스크린샷을 첨부해주세요. 또한 버그가 발생한 모습이 있는 포스트의 주소를 첨부해주시는 것도 아주 좋습니다!

**Comment by pronist** [comment, marked as answer]
@bbingse 안녕하세요.

먼저, 이 질문은 버그가 아니라 기능에 대한 질의이므로 **디스커션**으로 이전합니다.

이미지 지연로딩으로 이미지를 불러올 때 그 속도가 늦는 것은 인터넷 속도에 의존합니다.

기술적으로 해결이 가능할 수는 있겠으나 이를 수정하여 추가적으로 배포하지는 않습니다.

해결 방법이 없지는 않지만 순조롭지는 않습니다. 스킨에서 사용된 라이브러리는 [lazysizes](https://github.com/aFarkas/lazysizes) 이며 [Options](https://github.com/aFarkas/lazysizes?tab=readme-ov-file#js-api---options) 를 참고하여 설정을 재설정하고 스킨을 티도리 프레임워크를 사용해 재컴파일 해야 할 것입니다. 이 과정은 개발자가 아닌 경우 비효율적이므로 권장하지 않습니다.

그 이외에는 간편하고 편리한 선에서 제시해드릴 수 있는 방법이 없습니다.
